### PR TITLE
Enhancement/3103 account summaries

### PR DESCRIPTION
**Pull request by eugene-manuilov**
## Summary

Addresses issue #3103

## Checklist

- [x] My code is tested and passes existing unit tests.
- [x] My code has an appropriate set of unit tests which all pass.
- [x] My code is backward-compatible with WordPress 4.7 and PHP 5.6.
- [x] My code follows the [WordPress](https://make.wordpress.org/core/handbook/best-practices/coding-standards/) coding standards.
- [x] My code has proper inline documentation.
- [ ] I have added a QA Brief on the issue linked above.
- [x] I have signed the Contributor License Agreement (see <https://cla.developers.google.com/>).

### CODE REVIEW

File: assets/js/modules/analytics-4/datastore/accounts.test.js [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+	describe( 'selectors', () => {
+		describe( 'getAccountSummaries', () => {
+			it( 'should use a resolver to make a network request', async () => {
+				fetchMock.get( accountSummariesEndpoint, {
+					body: fixtures.accountSummaries,
+					status: 200,
+				} );
+
+				const initialProperties = registry.select( STORE_NAME ).getAccountSummaries();
+				expect( initialProperties ).toBeUndefined();
```
Comment (aaemnnosttv) [inline assets/js/modules/analytics-4/datastore/accounts.test.js:59]: Looks like this was probably copied from another test and there are a few variables which need to be renamed (this is just one example).

**Review by aaemnnosttv — CHANGES_REQUESTED** [review]
Looks good to me @eugene-manuilov – just a few things to fix up in the tests.

File: assets/js/modules/analytics-4/datastore/accounts.test.js (hunk 1 above)
Comment (eugene-manuilov) [reply assets/js/modules/analytics-4/datastore/accounts.test.js]: :facepalm: fixed!

**Review by aaemnnosttv — APPROVED** [review]
Great, thanks!

**Pull request merged**